eng/Versions.props

```diff
@@ -245,6 +245,7 @@
     <SystemDrawingCommonVersion>7.0.0</SystemDrawingCommonVersion>
     <SystemIOFileSystemVersion>4.3.0</SystemIOFileSystemVersion>
     <SystemIOFileSystemPrimitivesVersion>4.3.0</SystemIOFileSystemPrimitivesVersion>
+    <SystemIOHashingVersion>8.0.0-rc.2.23479.6</SystemIOHashingVersion>
```

Comment:
We won't want to ship a prerelease of this dependency with VS. Can you use the latest stable version instead?

Reply:
we cannot.   the latest stable does not include xxhash128 (the actual hashing function we're trying to use).